src/test/java/org/codewars/kata/SixTest.java

```diff
@@ -29,7 +29,7 @@ private Object[][] fTestData() {
         return UserImplementations.combineImplWithTests(UserImplementations.SIXES, testData);
 
     }
-    private static void assertFuzzyEquals(double act, double exp){
+    private void assertFuzzyEquals(double act, double exp){
```

Comment:
test annotation? + data source

Reply:
why this is necessary? It's just additional funticon for more accurate floating point measurement in the test below